Mathlib/GroupTheory/FixedPointFree.lean

```diff
@@ -34,7 +34,9 @@ end Definitions
 
 namespace FixedPointFree
 
--- todo: refactor Mathlib/Algebra/GroupPower/IterateHom to generalize φ to MonoidHomClass
+-- todo: refactor `Mathlib.Algebra.GroupPower.IterateHom` to generalize φ to MonoidHomClass
+-- Yaël: `Mathlib.Algebra.GroupPower.IterateHom` stopped existing in
+-- https://github.com/leanprover-community/mathlib4/pull/22132
```

Comment:
I think you should work out what the updated version of this comment is rather than making it more confusing.

Reply:
Sorry, I got rid of it entirely, but apparently committed to the wrong branch. Will fix